source/testing/acceptance.md

```diff
@@ -39,6 +39,37 @@ test('should add new post', function(assert) {
 });
 ```
 
+You can also stub a service into the acceptance test:
+```tests/acceptance/login-test.js
+import { test } from 'qunit';
+import moduleForAcceptance from 'people/tests/helpers/module-for-acceptance';
+
+const userStub = Ember.Service.extend({
+  email: 'test@testing.com',
+  token: '',
+  isLoggedIn() {
+    return true;
+  },
+});
+
+moduleForAcceptance('Acceptance | login', {
+  beforeEach(){
+    this.application.register('service:user-service', userStub);
```

Comment:
Hmm. I don't think this works correctly due to some shenanigans between `Ember.Application` instances and `Ember.ApplicationInstance` instances.  Basically, I do not believe that this mechanism will allow the stub to clobber and "win" over the actual module that would be resolved (`app/services/user-service.js` in this case).

The thing that I generally do is:

```js
this.register = (...args) => this.application.__deprecatedInstance__.register(...args);
this.lookup = (...args) => this.application.__deprecatedInstance__.lookup(...args);
```

Reply:
This is what the tutorial says. Perhaps we need to update the tutorial to the one you do?

Reply:
the tutorial seems to stub the services, but if that's not the suggested way, we can change it.  Write up an issue and/or pr and we can get it updated.

Reply:
I think stubbing the service is okay in acceptance test, but just we need to show people how to do it.